lib/brainstem/concerns/formattable.rb

```diff
@@ -0,0 +1,60 @@
+require 'brainstem/api_docs'
+require 'active_support/inflector/inflections'
+
+#
+# TODO: Spec this out
+#
+module Brainstem
+  module Concerns
+    module Formattable
```

Comment:
Nothing's done on inclusion or extension, and it doesn't mix class / instance methods, so nope, just a standard module should suffice!